pkg/lib/operation/template/sync/diff/create/operation.go

```diff
@@ -1,29 +1,21 @@
 package create
 
-import (
-	"context"
+// type Options struct {
```

Comment:
Why is this commented?

Reply:
This part of the code was unused. I don't know if it's ready for the future.

Reply:
Hm I would rather remove it instead of commenting it. If it is unsed probably does not need to be there at all. Put this part into your 1:1 with Michal to go through the `lib/operation` module what should be there and what not.